fiftyone/utils/hrm2.py

```
@@ -0,0 +1,2581 @@
+
+        Args:
+            batch_data: List of dicts from GetItem with 'image', 'detections', 'filepath'
+
+        Returns:
+            List of raw output dicts, one per image, each containing:
+                - "people": List[person_raw_dict] with raw tensors
+                - "img_shape": (height, width) of source image
+        """
+        raw_outputs = []
+
+        for idx, data in enumerate(batch_data):
```

Ruff (0.14.5)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)